### PR TITLE
Update 06_01_Systemized_Skills.txt

### PR DESCRIPTION
**Pull request by rosebrallier**
Added prose explanation to the skills document.
Organized skills by the most common applicable stat modifier.
Removed dev notes from the body of the document.
Edited Class Skill Proficiencies for consistency.

Made edits re: the following issues:
[270](https://github.com/bleehu/Compound_X/issues/270): Added jury-rig and weapons: explosives skills
[340](https://github.com/bleehu/Compound_X/issues/340): Defined Acrobatics as PARKOUR
[344](https://github.com/bleehu/Compound_X/issues/344): Limited skill point gain per level to 5, disregarding feats and class passives

### CODE REVIEW

**Comment by rosebrallier** [comment]
Nitpick: I could include picking mechanical locks in the sleight of hand description. Or make it it's own dex skill. Help I can't decide.

**Comment by Turtlelord26** [comment]
As someone with a fair amount of experience in both systems, DnD 3.5 had both Sleight of Hand and Open Lock, while Pathfinder just had Sleight of Hand. When I was playing Pathfinder, I didn't miss Open Lock.

(In depth review coming soon)

File: CharacterCreation/06_01_Systemized_Skills.txt
Comment (bleehu) [inline CharacterCreation/06_01_Systemized_Skills.txt]: Very good explanation! Clear, concise, all in one place! This is going to make onboarding new players _way_ easier! I didn't know how bad we needed this until I saw it.

File: CharacterCreation/06_01_Systemized_Skills.txt
Comment (bleehu) [inline CharacterCreation/06_01_Systemized_Skills.txt]: Non-blocking style nitpick: Can we capitalize "Proficient" in this context since we're using it as a game-term/proper noun and since it's in the Glossary? (since this is nonblocking, you can punt on this and merge this awesome load of work and add that later. Since it's a small nitpick and I'm asking instead of requesting you can also just say "naw, it's fine" and move on with the rest of the work you want to do)

File: CharacterCreation/06_01_Systemized_Skills.txt
Comment (bleehu) [inline CharacterCreation/06_01_Systemized_Skills.txt]: I really like this hard definition. Shipit. Shipit good. I especially like how it means that someone with no hacking skill can't break a DC 130 system even if they roll a natural 100, which always felt game breaking to me in the past.

File: CharacterCreation/06_01_Systemized_Skills.txt
Comment (bleehu) [inline CharacterCreation/06_01_Systemized_Skills.txt]: I really like how you took the time to explain this. I feel like that's a huge step toward someone being able to read our book and play the game without having to have someone from the existing community be there for their first few sessions to show people how it works.

File: CharacterCreation/06_01_Systemized_Skills.txt
Comment (bleehu) [inline CharacterCreation/06_01_Systemized_Skills.txt]: actually laughed out loud at reading this. :+1:

File: CharacterCreation/06_01_Systemized_Skills.txt
Comment (bleehu) [inline CharacterCreation/06_01_Systemized_Skills.txt:103]: When I first read this I thought of like... hell/hades/etc. After some brief confusion I realized it was like the criminal underworld. Maybe consider renaming to "streetsmarts" or something? But if I can tell what you meant just by reading, it doesn't need to be changed.

File: CharacterCreation/06_01_Systemized_Skills.txt
Comment (bleehu) [inline CharacterCreation/06_01_Systemized_Skills.txt]: nonblocking: I'd like a little more detail here. I'd be willing to add it myself. I want to explicitly call out that weapon skill helps with reloading, clearing jams, adding attachments in the field and goes along with certain weapon feats, but very specifically does not change accuracy rolls. I think this is the place to neutralize a very common confusion point.

Comment (bleehu) [reply CharacterCreation/06_01_Systemized_Skills.txt]: and you did a very good job creating the context for that opportunity to destroy confusion!

**Review by bleehu — APPROVED** [review]
Great work Rose! Especially impressed by your clear wording, zero style mistakes and incremental improvement on our product! Please hit that big green "Merge" button below and take a second to feel like a badass developer! Then hit the "Delete Branch" button after the Merge button, since the merge will move your work into the dev branch and we won't need this feature branch anymore! Way to go!

File: CharacterCreation/06_01_Systemized_Skills.txt
Comment (Turtlelord26) [reply CharacterCreation/06_01_Systemized_Skills.txt:103]: I am especially unconcerned about confusion with Hades since it's right next to Religion.

**Review by Reveraine — APPROVED** [review]
✓

File: CharacterCreation/06_01_Systemized_Skills.txt
Comment (Turtlelord26) [inline CharacterCreation/06_01_Systemized_Skills.txt]: We may wish an explicit reference to the leveling table in basic rules here.

File: CharacterCreation/06_01_Systemized_Skills.txt
Comment (Turtlelord26) [inline CharacterCreation/06_01_Systemized_Skills.txt]: I think we need a specific example inserted after this paragraph about class skill gain vs leveling skill gain cap. Specifically, if my class gives me 10 repair, can I also put 5 points into repair at level 1, going to 15, or do I have to wait until level 3, when the cap becomes 15, to add more points to Repair?

(I'm fairly confident you meant the former, but it should be spelled out.)

File: CharacterCreation/06_01_Systemized_Skills.txt
Comment (Turtlelord26) [inline CharacterCreation/06_01_Systemized_Skills.txt]: We should mention where those proficiencies come from, for better rules interreferencing. Backgrounds and classes, right?

File: CharacterCreation/06_01_Systemized_Skills.txt
Comment (Turtlelord26) [reply CharacterCreation/06_01_Systemized_Skills.txt]: Yes, a very good touch. And, as always, subject to individual DM discretion for when an auto-success would make sense.

File: CharacterCreation/06_01_Systemized_Skills.txt
Comment (Turtlelord26) [inline CharacterCreation/06_01_Systemized_Skills.txt]: Typography: "player can all all"

File: CharacterCreation/06_01_Systemized_Skills.txt
Comment (Turtlelord26) [inline CharacterCreation/06_01_Systemized_Skills.txt]: When dealing with progression systems, I always try to aim for maximum end-state power. This involves lots of turtling in RTS and 4X, for analogy. The following concern assumes that mindset is common enough to design around.

I am wary of ever putting points into General skills, because at some point General must become overshadowed by the Specific skills (and if it doesn't we've made a design error because then the specifics have no reason for existing). I wonder if we could mitigate that by having a threshold where having general skill helps your specific skill rolls. Something like having 30/60 points in the General lets you roll the average on a Specific check that you are not using your general score for (including in combat situations), and 60/60 rolls with advantage.

File: CharacterCreation/06_01_Systemized_Skills.txt
Comment (Turtlelord26) [reply CharacterCreation/06_01_Systemized_Skills.txt]: Though, in all seriousness, the end product probably wants a paragraph specifically dedicated to describing the different between Acrobatics and Athletics. I've seen a lot of really weird argument attempts from Dex-based characters trying to use acro for an athl task.

File: CharacterCreation/06_01_Systemized_Skills.txt
Comment (Turtlelord26) [inline CharacterCreation/06_01_Systemized_Skills.txt:74]: It seems more likely to me that players will be flying ships then helicopters. Can we have this be Space and collapse air vehicles into general instead?

File: CharacterCreation/06_01_Systemized_Skills.txt
Comment (Turtlelord26) [inline CharacterCreation/06_01_Systemized_Skills.txt:116]: Caring for a weapon is a skill, but not usually one that needs to be performed under pressure (reloading excepted). I'm having a hard time thinking of a situation where I'd need to roll a DC 100 Weapons - SMG check. Assuming I'm not suffering from a failure of imagination, can we consider completely combining these skills and Source's "Weapon proficiency feats" in the feat document?

**Review by Turtlelord26 — CHANGES_REQUESTED** [review]
Glorious. 🐢 
I have a couple clarity notes and some discussion notes probably destined to be shot out of this PR and into their own issues by Blee.

File: CharacterCreation/06_01_Systemized_Skills.txt
Comment (rosebrallier) [reply CharacterCreation/06_01_Systemized_Skills.txt]: I'm going to boot this discussion to a different pr and/or new issue.

File: CharacterCreation/06_01_Systemized_Skills.txt
Comment (rosebrallier) [reply CharacterCreation/06_01_Systemized_Skills.txt:74]: I disagree on this. Not only are there several dungeons with helicopters in them, I want to give room for other planet-side dungeons.
I'm willing to talk splitting air and space vehicles into separate skills, but not completely disregarding air vehicles as a specific case.
In conclusion, I'm going to boot this discussion to a different pr and/or new issue.

File: CharacterCreation/06_01_Systemized_Skills.txt
Comment (rosebrallier) [reply CharacterCreation/06_01_Systemized_Skills.txt]: Ja; I didn't specify it here because I haven't read the race documents fully to know if proficiencies come into play there.

File: CharacterCreation/06_01_Systemized_Skills.txt
Comment (bleehu) [reply CharacterCreation/06_01_Systemized_Skills.txt]: ^^^ good call, roseb. That is classic scope creep.

File: CharacterCreation/06_01_Systemized_Skills.txt
Comment (bleehu) [reply CharacterCreation/06_01_Systemized_Skills.txt:116]: Since they're still skills; I think it's appropriate to include them in the skills doc

Comment (Turtlelord26) [reply CharacterCreation/06_01_Systemized_Skills.txt:116]: Then the appropriate discussion concerns the skills hiding in the feats document. Will make a note.
Now lives in #398

File: CharacterCreation/06_01_Systemized_Skills.txt
Comment (Turtlelord26) [reply CharacterCreation/06_01_Systemized_Skills.txt:74]: Entirely possible I have a bias towards space exacerbated by never having participated in a dungeon with a helicopter.

File: CharacterCreation/06_01_Systemized_Skills.txt
Comment (rosebrallier) [reply CharacterCreation/06_01_Systemized_Skills.txt]: Done :)

File: CharacterCreation/06_01_Systemized_Skills.txt
Comment (rosebrallier) [reply CharacterCreation/06_01_Systemized_Skills.txt]: I've described acrobatics as applying to physical tasks that primarily require flexibility or quickness.

File: CharacterCreation/06_01_Systemized_Skills.txt
Comment (rosebrallier) [reply CharacterCreation/06_01_Systemized_Skills.txt]: Specified that the skill doesn't affect accuracy.

File: CharacterCreation/06_01_Systemized_Skills.txt
Comment (rosebrallier) [reply CharacterCreation/06_01_Systemized_Skills.txt]: Done.

File: CharacterCreation/06_01_Systemized_Skills.txt
Comment (rosebrallier) [reply CharacterCreation/06_01_Systemized_Skills.txt]: Clarified.

File: CharacterCreation/06_01_Systemized_Skills.txt
Comment (rosebrallier) [reply CharacterCreation/06_01_Systemized_Skills.txt]: Clarified.

File: CharacterCreation/06_01_Systemized_Skills.txt
Comment (rosebrallier) [reply CharacterCreation/06_01_Systemized_Skills.txt]: Done.

File: CharacterCreation/06_01_Systemized_Skills.txt
Comment (rosebrallier) [reply CharacterCreation/06_01_Systemized_Skills.txt:74]: If you want a Vehicles: Space skill, make an issue.

**Comment by Turtlelord26** [comment]
How rude of github to not let me approve the changes after it's merged.

**Pull request merged**
